.circleci/config.yml

```diff
@@ -1,8 +1,8 @@
 version: 2.1
 
 orbs:
-  codacy: codacy/base@10.11.2
-  codacy_plugins_test: codacy/plugins-test@1.1.1
+  codacy: codacy/base@11.1.2
```

Comment:
Latest version is 12.0.0
See: https://circleci.com/developer/orbs/orb/codacy/base